packages/ag-charts-website/src/content/docs/charts-nightingale-series/index.mdoc

```diff
@@ -0,0 +1,145 @@
+---
+title: "Nightingale Series"
+enterprise: true
+---
+
+**Nightingale** series (sometimes called a *Rose chart*) represent a dataset in a form of a polar area diagram.
+
+## Grouped Nightingale
+
+To plot a basic Nightingale we need an array of data, where every item will have a single category value (for the angle axis) and a single numeric value (for the radius axis).
+
+A minimal nightingale series configuration will look like:
+
+```js
+series: [{
+    type: 'nightingale',
+    angleKey: 'product',
+    radiusKey: 'sales',
+}]
+```
+
+{% chartExampleRunner title="Grouped Nightingale Chart" name="group-nightingale" type="generated" options="{ \"enterprise\": true }" /%}
+
+## Stacked Nightingale
+
+Nightingale sectors can be stacked over each other to form a total by each category. The next example demonstrates a stacked nightingale chart:
```

Comment:
In a stacked Nightingale series, sectors are placed on top of each other to create a total for each category. The following example illustrates a stacked Nightingale chart.